lib/sycamore/sycamore/tests/integration/connectors/opensearch/test_pdf_to_opensearch.py

```diff
@@ -115,12 +115,12 @@ def test_pdf_to_opensearch_with_llm_caching():
 
     try:
         context = sycamore.init(
-            config=Config(
-                opensearch_client_config=os_client_args,
-                opensearch_index_name="toyindex",
-                opensearch_index_settings=index_settings,
-                llm=openai_llm,
-            )
+            opensearch_config={
```

Comment:
This is where I thought it would be easier to write opensearch_config=OpenSearchConfig(os_client_args, "toyindex", index_settings)

If we had things that configure contexts having a context_varname property, then we could write this as just:
```
sycamore.init(openai_llm, OpenSearchArgs(os_client_args, "toyindex", index_settings)) 
```
and the positional args would be automatically put into kwargs via the context_varname.  Makes the code a little cleaner to use, unclear if it's too much magic.

Reply:
Yeah agree and that makes to me